src/parser.cpp

```diff
@@ -1318,26 +1318,26 @@ void Parser::ParserImpl::loadReset(const ResetPtr &reset, const ComponentPtr &co
             const std::string variableReference = attribute->value();
             VariablePtr referencedVariable = component->variable(variableReference);
 
-            // TODO This follows the same pattern as the errors returned from parsing the encapsulations.
+            // TODO This follows the same pattern as the issues returned from parsing the encapsulations.
```

Comment:
Not sure why this is a TODO seems more like a statement, if it's an issue create one this comment should be removed.

Reply:
Done in commit a1417d11.